source/slang/slang-ir.cpp

```diff
@@ -5043,6 +5132,8 @@ namespace Slang
         case kIROp_Nop:
         case kIROp_Specialize:
         case kIROp_lookup_interface_method:
+        case kIROp_RTTIExtractSize:
+        case kIROp_getAddr:
```

Comment:
Is `getAddr` still needed? I thought that with the explicit pointer-ification of things we could remove that because it was a bit of a hack...